resources/views/users/partials/_form_users.blade.php

```diff
@@ -0,0 +1,41 @@
+@include('layouts.errors')
+
+{{ csrf_field() }}
+
+<div class="form-item -padded">
+    <label class="field-label" for="key">User identification key:</label>
+    <input type="text" name="key" id="key" class="text-field" placeholder="Email, Mobile Phone Number, Northstar ID or Phoenix ID" value="{{ $user->id or old('key') }}"/>
+</div>
+
+<div class="form-item -padded">
+    <label class="field-label" for="type">Select user key type:</label>
+    <div class="select">
+        <select name="type">
+            <option value="email" {{! isset($user->id) ? 'selected' : '' }}>Email</option>
```

Comment:
@DFurnes Still a little iffy on dealing w/ preselected items in dropdowns and radios (below). Did this for now, but would prefer to be able to call a method, only issue is the passing of a non-set $variable...